### PR TITLE
ISSD-2738 [PRM] Improve MDF Request Table with new fields

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [ISSD-2738](https://liferay.atlassian.net/browse/ISSD-2738)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/interfaces/tableColumn.ts [hunk 1]
```diff
@@ -6,5 +6,6 @@
 export default interface TableColumn<T> {
 	columnKey: string;
 	label: string | JSX.Element;
+	label2?: string;
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/interfaces/tableColumn.ts:9]: The label prop accept JSX.Element, you should use this to create a dynamic label.

```suggestion
	label: string | JSX.Element;
```

**Comment by eliasantosf** [comment]
ci:test:sf

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestList/utils/getMDFListColumns.tsx [hunk 2]
```diff
@@ -29,6 +30,23 @@ export default function getMDFListColumns(
...
+			<div>
+				<p className="border-0 font-weight-normal mb-0 text-truncate text-truncate-inline">
+					{row[column1 as MDFColumnKey]}
+				</p>
+				<p className="mb-0 mt-0 text-neutral-7 text-paragraph-sm">
+					{row[column2 as MDFColumnKey]}
+				</p>
+			</div>
+		);
+	};
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestList/utils/getMDFListColumns.tsx:48]: Think about simplicity, I think you can pass this piece of code instead have this function

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestList/utils/getMDFListColumns.tsx [hunk 3]
```diff
@@ -154,50 +172,67 @@ export default function getMDFListColumns(
...
+			label: (
+				<div>
+					<p className="mb-0 mt-4 text-neutral-10">Submit Date</p>
+					<p className="mt-0 text-neutral-5 text-paragraph-sm">
+						Last Modified Date
+					</p>
+				</div>
+			),
+			render: (_, row) =>
+				getCustomColumn(row, 'DATE-SUBMITTED', 'LAST-MODIFIED'),
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestList/utils/getMDFListColumns.tsx:235]:
```suggestion
				<div>
				<p className="border-0 font-weight-normal mb-0 text-truncate text-truncate-inline">
					{row[column1 as MDFColumnKey]}
				</p>
				<p className="mb-0 mt-0 text-neutral-7 text-paragraph-sm">
					{row[column2 as MDFColumnKey]}
				</p>
			</div>
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by eliasantosf** [comment]
ci:test:sf

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by eliasantosf** [comment]
ci:close

**Pull request closed without merging**